forc-plugins/forc-client/src/util/tx.rs

```diff
@@ -59,6 +64,9 @@ fn prompt_signature(tx_id: fuel_tx::Bytes32) -> Result<Signature> {
 fn ask_user_yes_no_question(question: &str) -> Result<bool> {
     print!("{question}");
     std::io::stdout().flush()?;
+    if option_env!("CLI_TEST").is_some() {
+        return Ok(true);
+    }
```

Comment:
Not required at all but might make sense to create a macro that expands to this, as this appears a lot. Would bring most of these checks to single line from 3-4 lines.

`execute_if_cli_test!(return Ok(true)))`

Reply:
Also I am a little out of the loop so this might be already discussed. What was the reason we are doing these with `if` and not `cfg`s?

Reply:
Arent' these only relevant while we are running `cargo test`? what would be stopping us from using something like `cfg(test)` instead of a seperate env variable `CLI_TESTS`. Do we want to be able to run tests without cli tests or something?

Reply:
We can maybe make `if_cli_tests { x }` macro to expand into:

```rust
#[cfg(test)]
{
x
}
```

Reply:
Ohh I see if we have other unit tests touching those functions things might get complicated, you may want to test the rest of the function but cli tests insertion would return early etc. A final thought is to make this into a separate flag:

```rust
fn test_me() {
    #[cfg(cli_test)]
    {
    a = 10;
    }
}
```

If you do not want to setup features in cargo tomls, you can set thorough env variables:

```console
RUSTFLAGS="--cfg cli_test" cargo test
```

Reply:
Cool